projects/core/src/occ/adapters/product/default-occ-product-config.ts

```diff
@@ -22,6 +22,9 @@ export const defaultOccProductConfig: OccConfig = {
           attributes: 'products/${productCode}?fields=classifications',
           price: 'products/${productCode}?fields=price(formattedValue)',
           stock: 'products/${productCode}?fields=stock(DEFAULT)',
+          unit: 'products/${productCode}?fields=sapUnit',
+          productAvailabilities:
```

Comment:
MAJOR: Since the schema of the response from /productAvailabilites is not the same (or a subset) of the schema of the /products endpoint, lets create a new end point for /productAvailabilites rather than including it in product config

Reply:
Done